src/main/kotlin/no/nav/familie/ef/sak/brev/BrevMellomlagerController.kt

```diff
@@ -44,32 +41,6 @@ class BrevMellomlagerController(
         )
     }
 
-    @Deprecated("Skal slettes")
-    @PostMapping("/frittstaende")
-    fun mellomlagreFrittstaendeBrev(@RequestBody mellomlagretBrev: FrittståendeBrevDto): Ressurs<UUID> {
-        tilgangService.validerTilgangTilFagsak(mellomlagretBrev.fagsakId, AuditLoggerEvent.UPDATE)
-        tilgangService.validerHarSaksbehandlerrolle()
-
-        return Ressurs.success(mellomlagringBrevService.mellomlagreFrittståendeBrev(mellomlagretBrev))
-    }
-
-    @Deprecated("Skal slettes")
-    @DeleteMapping("/frittstaende/{fagsakId}")
-    fun mellomlagreFrittstaendeBrev(@PathVariable fagsakId: UUID): Ressurs<UUID> {
-        tilgangService.validerTilgangTilFagsak(fagsakId, AuditLoggerEvent.DELETE)
-        tilgangService.validerHarSaksbehandlerrolle()
-        mellomlagringBrevService.slettMellomlagretFrittståendeBrev(fagsakId, SikkerhetContext.hentSaksbehandler())
-        return Ressurs.success(fagsakId)
-    }
-
-    @Deprecated("Skal slettes")
-    @GetMapping("/frittstaende/{fagsakId}")
```

Comment:
Denne endepunkten er fortsatt i bruk, burde kode i frontend fjernes først?

Reply:
@PostMapping("/send") har også blitt brukt
Kanskje det også som gjør at det ser ut som at siste mellomlagrede brevet er fra juni, fordi funksjonaliteten for å sende friståeende brev også fjerner mellomlagret brev?

Reply:
Ja, frontende må fikses først. Fint du følger med! https://github.com/navikt/familie-ef-sak-frontend/pull/2485